index.html

```diff
@@ -126,7 +164,7 @@
               ><label for="CSS">CSS</label
               ><label for="Ruby">Ruby and Rails</label>
             </div>
```

Comment:
This is a list of items, so kindly put them in a ul tag instead of div and use an li tag for each of them. 👍